locales/en/common.json

```diff
@@ -0,0 +1,13 @@
+{
+  "CONNECT_WALLET_BUTTON": "Connect wallet",
+  "BUTTON_SUBMITTING": "Submitting...",
+  "BUTTON_CANCEL": "Cancel",
+
+  "LAYOUT_HEADING": "Nengajo – HENKAKU",
+  "HOME_LINK": "Home",
+  "COLLECTION_LINK": "Collections",
+  "REGISTER_LINK": "Register",
```

Comment:
How about "Register a Nengajo" and "年賀状を登録"?

I think using just "register" is OK on code, but on UI it can sound like you would be registering as a user.

Reply:
From the meeting:

Create / 作る

Reply:
Wasn't it like this?
Create Nengajo / 年賀状を作る

Reply:
Sorry. I must have taken the wrong note. Let's do that 🙇